src/core/annotation.js

```diff
@@ -1125,7 +1125,12 @@ class Annotation {
       }
 
       if (loopDict.has("T")) {
-        fieldName.unshift(stringToPDFString(loopDict.get("T")));
+        const t = stringToPDFString(loopDict.get("T"));
+        if (!t.startsWith("#")) {
```

Comment:
Does this apply to the `T`-entry at line 1100 above as well, or only here?

Reply:
A final field isn't a container, hence it should have a name.